config/routes.rb

```diff
@@ -0,0 +1,12 @@
+# frozen_string_literal: true
+
+Rails.application.routes.draw do
+  resources :categories do
+    resources :payments
```

Comment:
- [ ] Kindly expose the routes in use such as index, show, new, etc, except put/update.

Reply:
Resolved